src/realm/list.hpp

```diff
@@ -427,6 +427,9 @@ class Lst<Mixed> final : public CollectionBaseImpl<LstBase>, public CollectionPa
         insert(size(), std::move(value));
     }
 
+    template <typename T>
+    void add_json(const T&);
```

Comment:
Minor stuff: Shouldn't this be `insert_json` like for dictionaries? Probably, I would prefer to have the same aligned interface both for `List` and `Dictionary` if I was using this.

Reply:
`insert` would require a position to insert at, which seems not to be required for this use.